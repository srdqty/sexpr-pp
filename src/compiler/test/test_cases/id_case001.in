(lambda (x y) x y)
(lambda (x y) (lambda (x z) x y z))
