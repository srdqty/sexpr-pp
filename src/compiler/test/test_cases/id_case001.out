(FunFixArity (Arity = 2) (ShallowRef 0) (ShallowRef 1))
(FunFixArity (Arity = 2) (FunFixArity (Arity = 2) (ShallowRef 0) (DeepRef 1 1) (ShallowRef 1)))
